FHICORC.Core/Services/Model/SmartHealthCardModel/Shc/SmartHealthCardObservation.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using FHICORC.Core.Services.Model.Converter;
+using Newtonsoft.Json;
+
+namespace FHICORC.Core.Services.Model.SmartHealthCardModel.Shc
+{
+    public class SmartHealthCardObservation
+    {
+        [JsonProperty("status")]
+        public string Status { get; set; }
+
+        [JsonProperty("code")]
+        public CodeableConcept TestCode { get; set; }
+
+        [JsonProperty("subject")]
+        public Subject Subject { get; set; }
+
+        [JsonProperty("effectiveDateTime")]
+        [JsonConverter(typeof(ISO8601DateTimeConverter))]
+        public DateTime? EffectiveDateTime { get; set; }
+    }
```

Comment:
I think we need to include the test result, defined in _valueCodeableConcept_, _valueQuantity_ or _valueString_ (1 of 3 is required). [Documentation](https://build.fhir.org/ig/HL7/fhir-shc-vaccination-ig/StructureDefinition-shc-covid19-laboratory-result-observation-dm.html).
@NC-395  Dont know if we need to support all these, this [example](https://raw.githubusercontent.com/HL7/fhir-shc-vaccination-ig/master/examples/Scenario3Bundle-jws-payload-expanded.json) uses _valueCodeableConcept_

Reply:
We should do it according to the spec, if it says one of three is required, we should include all three as nullables, however, I do not see how `valueQuantity` can be used in context of COVID-19 test result, which is either DETECTED/NOT DETECTED. `valueString` is something, I guess, issuers can use for displaying COVID-19 test result, e.g., NEGATIVE. But if this is a string that can pretty much include anything, I am not sure how much freedom we have when using it + there is a lot of uncertainties.

@NC-001 Maybe add them as nullables to the model, I expect in 99% of cases we will see a CodableConcept.